tasks/main.yml

```diff
@@ -1 +1,40 @@
 # role tasks
+
+- block:
+  - name: install NFS server packages
+    package:
+      name: "{{ item }}"
+      state: present
+    loop: "{{ nfs_server_packages }}"
+
+  - name: optionally create share directories
+    file:
+      path: "{{ item.share_path }}"
+      state: directory
+      owner: "{{ item.owner | default(omit) }}"
+      group: "{{ item.group | default(omit) }}"
+      mode: "{{ item.mode | default(omit) }}"
+    loop: "{{ nfs_server_shares }}"
```

Comment:
`loop` was introduced in Ansible 2.5, so either the dependency needs to be updated to Ansible 2.5+, or this should be `with_items`. Users of 2.4 will probably see a message along the lines of `ERROR! The field 'loop' is supposed to be a string type, however the incoming data structure is a <class 'ansible.parsing.yaml.objects.AnsibleSequence'>`